README.md

```diff
@@ -589,7 +628,9 @@ git clone https://github.com/AsimovPlatform/protoflow.git
 [`ReadDir`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadDir.html
 [`ReadEnv`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadEnv.html
 [`ReadFile`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadFile.html
+[`ReadFile`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadSocket.html
```

Comment:
There's a typo here: should be `ReadSocket`, but it says `ReadFile`

Reply:
fixed